src/routes/tickets/util/getMetadataFromTicket.ts

```diff
@@ -1,11 +1,29 @@
 import { ITicket, ITicketMetaData } from '../controllers/ticketController';
 
+
 export const getMetadataFromTicket = (data: ITicket[]): ITicketMetaData[] => {
-	const metadata = data;
+  const metadata = data;
+  let flattenedMessage = '';
+
+  // Take all the messages put them together into a single string and if the string is equals or grater than 50 return. Once 50 add three dots at the end of te string and return that as previewString
```

Comment:
The code doesn't match this comment. The code currently takes each text paragraph in the first message and flatmaps it.

Reply:
Well, then rest does what's commented :)

Reply:
Nvm